README.md

```diff
@@ -37,7 +37,6 @@ meminfo | Exposes memory statistics. | Darwin, Dragonfly, FreeBSD, Linux
 netdev | Exposes network interface statistics such as bytes transferred. | Darwin, Dragonfly, FreeBSD, Linux, OpenBSD
 netstat | Exposes network statistics from `/proc/net/netstat`. This is the same information as `netstat -s`. | Linux
 sockstat | Exposes various statistics from `/proc/net/sockstat`. | Linux
-stat | Exposes various statistics from `/proc/stat`. This includes CPU usage, boot time, forks and interrupts. | Linux
```

Comment:
Bit confusing that to get the boot time and some other CPU-unrelated stats, you now have to enable the CPU collector? But I also don't have a great answer to this question, maybe that's just the way it is...